pkg/pfconfig/basicstationlns/basicstationlns.go

```diff
@@ -116,10 +116,13 @@ func GetRouterConfig(bandID string, fps map[string]*frequencyplans.FrequencyPlan
 		// These fields are not defined in the v1.5 ref design https://doc.sm.tc/station/gw_v1.5.html#rfconf-object and would cause a parsing error.
 		sx1301Conf.Radios[0].TxFreqMin = 0
 		sx1301Conf.Radios[0].TxFreqMax = 0
-		// Remove hardware specific values that are not necessary.
+		// Remove hardware specific values that are handled by the gateway itself.
+		sx1301Conf.ClockSource = nil
```

Comment:
Isn't the underlying problem that we have radio specific fields in the frequency plan?

Reply:
Well these fields are valid `SX1301_conf` fields. Issue is that basic station specifically already has a config file on each hardware and hardware specific fields are already present there.

Reply:
But isn't that true for other gateways too?

Reply:
Most gateways use what's provided in the `global_conf.json` right? Some have overrides in `local_conf.json` but not all of them afaik.

Reply:
What I understand from #2130 is that we send information too specific about a gateway to a gateway. That makes me think that we have fields in our frequency plans that don't belong there. So removing those from BSLNS as you do here is a fix, but it's a fix for the symptoms. We may need to remove these fields from the frequency plans, `pfconfig` and GCS.